host.conf

```diff
@@ -0,0 +1,26 @@
+server {
+    listen               443 ssl;
+    ssl_certificate      /etc/nginx/my-site.com.crt;
+    ssl_certificate_key  /etc/nginx/my-site.com.key;
```

Comment:
Are these self signed certs? Also, it is not in this commit so it would fail. Maybe we just use http only

Reply:
It's not on commit because you need to generate your own self certificates as per I included on readme.md instruction and not upload my mine (security reasons).

Reply:
Some ppl will have hard time to generate self signed certs. I think we just stick on http

Reply:
I'll comment ou these lines and update the instructions on readme.md, could be in that way?

Because some payment methods to we work in localhost we need has an https connection to their end work with us.

Or you think I should upload my own self-certificate?

Reply:
I see. But I am not sure how you can go about it.